tsdb/tsm1/file_store.go

```diff
@@ -339,22 +343,29 @@ func (t *fileTracker) AddBytes(bytes uint64, level int) {
 	atomic.AddUint64(&t.diskBytes, bytes)
 
 	labels := t.Labels()
+	if level > 5 {
+		level = 5
+	}
 	labels["level"] = fmt.Sprintf("%d", level)
 	t.metrics.DiskSize.With(labels).Add(float64(bytes))
 }
 
 // SetFileCount sets the number of files in the FileStore.
 func (t *fileTracker) SetFileCount(files map[int]uint64) {
 	labels := t.Labels()
-	level := uint64(0)
+	maxLevel := uint64(0)
 	for k, v := range files {
-		labels["level"] = fmt.Sprintf("%d", k)
-		if uint64(k) > level {
-			level = uint64(k)
+		level := uint64(k)
+		if level > 5 {
+			level = 5
+		}
+		labels["level"] = fmt.Sprintf("%d", level)
```

Comment:
If the level is 5 or over then the label should be `5+` really.

Reply:
nit: `6+` since it's `if level > 5`